docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
+services:
+  db:
+    image: postgres
```

Comment:
Do you need to specify the postgres docker environment variables?
I think it would work if they were already in your shell.